ansible/roles/geotrellis.libgdal-java/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Update the .bashrc to set JAVA_GDAL_DIR so that GeoTrellis can find GDAL bindings
+  lineinfile:
+    dest=/home/vagrant/.bashrc
```

Comment:
That's a good point - there's another environment setting that should be updated too in the spark role wrapper playbook